internal/common/mqtt/mqttconnection.go

```diff
@@ -0,0 +1,80 @@
+package mqtt
```

Comment:
Need a copyright and license statement at the head of file here.

Reply:
Done!